### PR TITLE
chore: update readme and examples

### PR DESCRIPTION
**Pull request by oliverpauffley**
Add description of how to generate certificates

### CODE REVIEW

File: README.md
Comment (rcrowe) [inline README.md:36]: Might be worth calling this out as 2 years, perhaps?

Comment (oliverpauffley) [reply README.md:36]: Yeah its one of those things that you need to understand not just copy. I will add something for that

**Comment by rcrowe** [comment]
https://github.com/utilitywarehouse/partner/blob/master/run-books/cockroach-db/certificate-authority.md?plain=1#L35 process for renewing an expiring token; I know this is in addition to your PR but the content matches & might be good to slip this also in here or another PR

**Review by rcrowe — APPROVED** [review]
👏🏻 🚀 😍 

Thank you

**Review by rcrowe — APPROVED** [review]

File: README.md
Comment (sbuliarca) [inline README.md:24]: Reading this the first time, I think this section would be better placed under a `Setup certificate authority` heading.

Also I think it would be nice to setup a common base for the CA too, and this part would sit in there. But until we get there, these kind of updates are more than welcome.

Comment (oliverpauffley) [reply README.md:24]: Yeah that was a another stumbling block when working on this. I could split this into two files linked in the readme? Then when we do split the repo this is slightly easier?

Comment (sbuliarca) [reply README.md:24]: Indeed, that's a great idea

**Comment by oliverpauffley** [comment]
Sorry @rcrowe @sbuliarca I went to town. 

Really what I did when I deployed this was follow the steps in two readmes then used the examples I found in a different namespace. So what I have done is move all of that into here. Its now a lot more hand-holdy but maybe this is better? Let me know

**Comment by sbuliarca** [comment]
This is so awesome. Thanks for handling it!

**Review by sbuliarca — APPROVED** [review]

**Comment by oliverpauffley** [comment]
@sbuliarca can you merge this when you are happy since I dont have write access

**Comment by sbuliarca** [comment]
> @sbuliarca can you merge this when you are happy since I dont have write access
Seems I don't have access either. @rcrowe ?

**Pull request merged**
